homeassistant/components/rachio.py

```diff
@@ -13,7 +13,7 @@
 from homeassistant.auth.util import generate_secret
 from homeassistant.components.http import HomeAssistantView
 from homeassistant.const import CONF_API_KEY, EVENT_HOMEASSISTANT_STOP, URL_API
-import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers import discovery, config_validation as cv
```

Comment:
Oh man discovery, all the cool kids are already using config entries!

Reply:
That's the next step! Minimal features per PR!

(really I just needed unique_id here, and fixed this while I had it open)